crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -497,7 +499,31 @@ pub mod error {
         InvalidClearingPrices,
         #[error(transparent)]
         Math(#[from] Math),
+        #[error("failed to calculate custom prices")]
+        CalculateCustomPrices(#[source] Trade),
+        #[error("missing native price for token {0:?}")]
+        MissingPrice(TokenAddress),
+        #[error("failed to calculate custom price {0:?}")]
```

Comment:
What makes this error different from `CalculateCustomPrices`? From the perspective of the solver they seem pretty indistinguishable and not self explanatory.

Reply:
Fair enough, that error could be squashed into its upper level error `error::Scoring` without a problem.